authors/apps/articles/views/articles.py

```diff
@@ -193,3 +195,62 @@ class TagView(ListAPIView):
     renderer_classes = (TagJSONRenderer,)
     serializer_class = CustomTagSerializer
     queryset = Tag.objects.all()
+
+
+class ArticleFilter(filters.FilterSet):
```

Comment:
To optimize our imports what do you think if we may just import the FilterSet class from filters instead of the whole of filters? I think there was that kind of reasoning when we were dealing with generic views a few weeks ago. What do you think?

Reply:
I get it